libs/paths/Ppath.ml

```diff
@@ -324,7 +357,7 @@ let in_project_unsafe_for_tests ~(phys_root : Fpath.t) (path : Fpath.t) =
            "cannot make path %S relative to project root %S.\n\
             cwd: %s\n\
             realpath for .: %s\n\
-            Sys.argv: %s" !!path !!phys_root (Sys.getcwd ())
+            Sys.argv: %s" !!path !!phys_root (Unix.getcwd ())
```

Comment:
I tested on a windows machine and both `Sys.getcwd ()` and `Unix.getcwd ()` return the same result (uppercase drive letter).